src/pages/home/report/ReportActionsList.tsx

```diff
@@ -60,6 +60,9 @@ type ReportActionsListProps = {
     /** Sorted actions prepared for display */
     sortedReportActions: OnyxTypes.ReportAction[];
 
+    /** Sorted actions that is visible to the user */
```

Comment:
The comments of `sortedReportActions` and `sortedVisibleReportActions` are a bit confusing. What's the difference between "prepared for display" and "visible to the user"?

Reply:
And why do we pass both of them as props? Do we need both?

Reply:
And why do we pass both of them as props? Do we need both?

Reply:
Yes, `sortedReportActions` is still being used inside it. `sortedReportActions` contains all the actions, including the deleted one.

Reply:
Ok got it. Then for `sortedReportActions` I would suggest something like: 
> All report actions, sorted by date

And then for `sortedVisibleReportActions` I would suggest something like: 
> Report actions that should be visible to the user, sorted by date

Reply:
Oh sorry, actually the `reportActions` isn't neccesarrily all the report actions of a report. It has much more complex logic to construct the list.

https://github.com/Expensify/App/blob/a60985ded98bc44f9c99f67d83b87e123d9543ac/src/pages/home/report/ReportActionsView.tsx#L233-L249
https://github.com/Expensify/App/blob/a60985ded98bc44f9c99f67d83b87e123d9543ac/src/pages/home/report/ReportActionsView.tsx#L151-L209

So, I think `Sorted actions prepared for display` comment kinda make sense? 🤔

Reply:
Ahh I see. Ok, I'm good with leaving as-is then.